pom.xml

```diff
@@ -378,6 +379,7 @@
         <jaxbutil.version>2.1.0</jaxbutil.version>
         <netty.version>4.1.105.Final</netty.version>
         <slf4j.version>2.0.9</slf4j.version>
+        <titanium.version>1.4.0</titanium.version>
```

Comment:
this titanium library is what jena 5 is using, but jena 5 can't take full advantage yet, so i've added it as dependency to be used directly